day_7/high_level.md

```diff
@@ -0,0 +1,32 @@
+## Caesar Cipher
+
+This is the order in which I worked through how to solve this problem. I worked through it in an "inside out" manner.
+
+1. I know I need to be able to iterate through each letter of a string in order to shift the string so the first step is to find a way to convert the desired string into an array. I found `array = string.char` would accomplish this.
+2. Next I will need a block of code that will iterate through each character, rotate it `x` number of places in the alphabet, and then return the new array. This would be in the form of:
+```ruby
+array.each do |char|
+  #code that will rotate letters and return new array with new letters
+end
+```
+There are likely a number of different ways to accomplish this, but the method I came up with, and what seemed most logical, was to use a hash and make the keys letters and the values numbers. In this case, each `char` could be passed through the hash as a key which would correspond to a certain integer value. Then the character that was returned would be the character corresponding to the hash's integer value plus or minus `x`.
```

Comment:
There are many ways to accomplish this. Also consider how arrays could help you keep your alphabet organized and accessible through index numbers.